rpc/ethereum/pubsub/pubsub.go

```diff
@@ -133,8 +133,8 @@ func (m *memEventBus) closeAllSubscribers(name string) {
 
 func (m *memEventBus) publishAllSubscribers(name string, msg coretypes.ResultEvent) {
 	m.subscribersMux.RLock()
+	defer m.subscribersMux.RUnlock()
 	subsribers := m.subscribers[name]
```

Comment:
nit: could be renamed to `subscribers`
```suggestion
	subscribers := m.subscribers[name]
```